test/controllers/versions_controller_test.rb

```diff
@@ -0,0 +1,8 @@
+require "test_helper"
+
+class VersionControllerTest < ActionDispatch::IntegrationTest
+  test "GET show should be successful" do
+    get "/check.svg"
```

Comment:
@jvanbaarsen you can parse the response body with an assert. And since SVG is just text, you can assert something that appears inside the SVG that you know will be there.